x/auth/vesting/msg_server.go

```diff
@@ -121,6 +121,11 @@ func (s msgServer) CreatePeriodicVestingAccount(goCtx context.Context, msg *type
 	if acc := ak.GetAccount(ctx, to); acc != nil {
 		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "account %s already exists", msg.ToAddress)
 	}
+
+	if msg.StartTime < ctx.BlockTime().Unix() {
```

Comment:
Eliminate this check. We would like to be able to create accounts which should already be vesting. I think the check for the vested amount works just fine with arbitrary start timestamp.